manage-data/lifecycle/index-lifecycle-management/manage-lifecycle-integrations-data.md

```diff
@@ -0,0 +1,64 @@
+
+1. Open **Dashboards** from the {{kib}} menu and select a dashboard to view. For example, with the [System integration](integration-docs://reference/system.md) installed, you can open the `[Metrics System] Host overview` dashboard to find visualizations about the host system being monitored.
+
+1. Hover over any visualization and click the **Explore in Discover** icon.
+
+    ![Explore in discover](/manage-data/images/ilm-explore-in-discover.png "")
+
+1. In **Discover**, the list of documents shows the columns for fields applicable to the visualzation that you selected. Select any document that has data for those fields and click the **Toggle dialog with details** icon.
+
+    ![Discover documents list](/manage-data/images/ilm-toggle-document-details.png "")
+
+1. In the document details, note that there are three `data_stream` fields. The full [data stream name](/reference/fleet/data-streams.md#data-streams-naming-scheme) is a composite of `data_stream.type`, `data_stream.dataset` and `data_stream.namespace`, separated by a hyphen. For example, in the System integration, the **CPU usage over time** visualization is associated with the `metrics-system.cpu-default` data stream.
+
+    You can also see the data stream's current backing index, as well as other information such as the document timestamp and details about the agent that ingested the data.
+
+    ![Document details](/manage-data/images/ilm-document-data-stream.png "")
+::::
+
+::::{dropdown} Find the data streams for an integration
+To find the data streams associated with an installed integration:
+
+1. In {{kib}} go to **Management > Integrations > Installed integrations** and select any integration.
+
+1. Open the **Assets** tab and expand the **Index templates** list.
+
+   In the list, the name of each [index template](/manage-data/data-store/templates.md) matches an associated data stream. For example, the `metrics-system.cpu` template matches the `metrics-system.cpu-default` data stream that is set up when the System integration is installed.
+
+    ![Integration assets](/manage-data/images/ilm-integration-assets.png "")
+::::
+
+::::{dropdown} Find the data streams managed in {{fleet}}
+To find all of the data streams that are managed in {{fleet}}:
+
+1. In {{kib}} go to **Management > {{fleet}}** and select the **Data streams** tab.
+
+1. Use the search field and dropdown menus to filter the list. You can filter by the data stream type, dataset, namespace, or by the integration that the data stream belongs to.
+
+    ![Integration assets](/manage-data/images/ilm-fleet-data-streams.png "")
+::::
+
+For any data stream that you're interested in, you can [view its current lifecycle status](/manage-data/lifecycle/index-lifecycle-management/policy-view-status.md), including details about its associated ILM policy. 
+
+Then, when you've identified one or more data streams for which you'd like to customize how the data is managed over time, refer to our tutorials:
```

Comment:
My intention was to suggest a simplification, but it may have made the structure more complicated :bowtie:  -- in which case feel free to ignore it.

```suggestion
After you've identified one or more data streams you want to customize, refer to these tutorials to adjust any of lifecycle settings that influence how your data is managed over time:
```